stdlib/messaging/nats/src/main/java/org/ballerinalang/nats/streaming/consumer/Attach.java

```diff
@@ -41,8 +42,15 @@ public static void streamingAttach(ObjectValue streamingListener, ObjectValue se
         ConcurrentHashMap<ObjectValue, StreamingListener> serviceListenerMap =
                 (ConcurrentHashMap<ObjectValue, StreamingListener>) streamingListener
                         .getNativeData(STREAMING_DISPATCHER_LIST);
-        serviceListenerMap.put(service, new StreamingListener(service, BRuntime.getCurrentRuntime(),
-                                                              streamingListener.getObjectValue("connection")
-                                                                      .getStringValue(Constants.URL)));
+        boolean manualAck = getAckMode(service);
+        serviceListenerMap.put(service, new StreamingListener(service, manualAck, BRuntime.getCurrentRuntime(),
```

Comment:
This is not formatted according to the standard. Previous was correct.

Reply:
Fixed this in 128133cde87cddcd31fc07a47ca7dada6e83fed0